src/mailer.py

```diff
@@ -1,11 +1,38 @@
-import smtplib, ssl
+import smtplib
+
+def send_mail(recipient, subject, body):
+  # Sender email and password
+  user = 'jojo@vg.no'
+  password = '1234'
+
+  # Connect to the Gmail SMTP server
+  smtp_server = smtplib.SMTP_SSL('smtp.gmail.com', 465)
+
+  # Login to the Gmail server
+  smtp_server.login(user, password)
+
+  # Construct the email message
+  message = f'Subject: {subject}\n\n{body}'
+
+  # Send the email
+  smtp_server.sendmail(user, recipient, message)
+
+  # Disconnect from the server
+  smtp_server.quit()
 
 def send_success_email(job, connection, result, *args, **kwargs):
-    ssl_context = ssl.create_default_context()
+  email = job.meta.get('email')
+  subject = 'Your file is ready'
+
+  download_url = "http://127.0.0.1:3000/v1/download/" + job.id
+  body = f'Your file is ready. Download it here: {download_url}'
```

Comment:
The body should be multi line so they can download it with the different formats

http://127.0.0.1:3000/v1/download/abc123/?output=vtt
http://127.0.0.1:3000/v1/download/abc123/?output=srt
http://127.0.0.1:3000/v1/download/abc123/?output= json
http://127.0.0.1:3000/v1/download/abc123/?output=txt